llama-index-integrations/llms/llama-index-llms-bedrock/llama_index/llms/bedrock/utils.py

```diff
@@ -85,6 +109,18 @@
     "mistral.mixtral-8x7b-instruct-v0:1",
     "mistral.mistral-large-2402-v1:0",
     "mistral.mistral-large-2407-v1:0",
+    "us.anthropic.claude-3-haiku-20240307-v1:0",
+    "us.anthropic.claude-3-opus-20240229-v1:0",
+    "us.anthropic.claude-3-sonnet-20240229-v1:0",
+    "us.anthropic.claude-3-5-haiku-20241022-v1:0",
+    "us.anthropic.claude-3-5-sonnet-20240620-v1:0",
+    "us.anthropic.claude-3-5-sonnet-20241022-v2:0",
+    "eu.anthropic.claude-3-haiku-20240307-v1:0",
+    "eu.anthropic.claude-3-opus-20240229-v1:0",
+    "eu.anthropic.claude-3-sonnet-20240229-v1:0",
+    "eu.anthropic.claude-3-5-haiku-20241022-v1:0",
+    "eu.anthropic.claude-3-5-sonnet-20240620-v1:0",
+    "eu.anthropic.claude-3-5-sonnet-20241022-v2:0",
```

Comment:
omg 😓 There has to be a better way to manage these names lmao

Reply:
I feel the same way! I can try refactoring the code to help with this, but this was the quickest way I found to make it work.